### PR TITLE
fix incorrect unbond_amount

### PR DESCRIPTION
**Pull request by simonjiao**
* **make sure that you have executed all the following process and no errors occur**
  - [x] make fmt
  - [x] make lint
  - [x] make test

* **The major changes of this PR**
fix incorrect bond_amount in DelegationInfo

* **Extra documentations**

* **Reletad issue**
close FindoraNetwork/platform-readonly#549

### CODE REVIEW

**Comment by simonjiao** [comment]
unstake 10000
unstake 5000
paid 10000
paid 5000

File: src/ledger/src/staking/mod.rs [hunk 1]
```diff
@@ -857,6 +862,16 @@ impl Staking {
...
                 .get_mut(h)
                 .map(|addrs| addrs.remove(addr));
+
+            // If this is a temporary delegation, remove it from original one
+            if let Some(receiver) = d.receiver_pk {
+                let orig_d = self
+                    .di
+                    .addr_map
+                    .get_mut(&receiver)
+                    .c(d!("Cannot find original delegation"))?;
```
Comment (ktmlm) [inline src/ledger/src/staking/mod.rs:872]: may be not an error in this scene, so using `if let ... get_mut()` will be better

**Review by ktmlm — CHANGES_REQUESTED** [review]
- use `tmp_delegators` instead of `temp_delegators` to express 'temporary' (avoid ambiguity with 'template')

File: src/ledger/src/staking/mod.rs (hunk 1 above)
Comment (simonjiao) [reply src/ledger/src/staking/mod.rs]: ok

**Review by ktmlm — APPROVED** [review]

**Pull request merged**